src/batou_ext/resources/oci-template.nix

```diff
@@ -17,10 +17,14 @@
   virtualisation.oci-containers = {
     backend = "docker";
     containers."{{component.container_name}}" = {
-      # {% if component.entrypoint != None %}
+      # {% if component.entrypoint %}
       entrypoint = "{{component.entrypoint}}";
       # {% endif %}
 
+      # {% if component.docker_cmd %}
+      cmd = [ {% for cmd in component.docker_cmd.split(" ") %} "{{cmd}}" {% endfor %} ];
```

Comment:
I'm pretty sure this will fail sooner or later. You probably want to use `shlex.split()`

Reply:
that's a very good point, just added that functionality